src/main/kotlin/no/nav/syfo/identhendelse/IdenthendelseService.kt

```diff
@@ -16,16 +17,16 @@ class IdenthendelseService(
 ) {
     private val log: Logger = LoggerFactory.getLogger(IdenthendelseService::class.java)
 
-    suspend fun handle(identhendelse: KafkaIdenthendelseDTO) {
+    fun handle(identhendelse: KafkaIdenthendelseDTO) {
         if (identhendelse.folkeregisterIdenter.size > 1) {
             val activeIdent = identhendelse.getActivePersonident()
             if (activeIdent != null) {
                 val inactiveIdenter = identhendelse.getInactivePersonidenter()
-                val aktivitetskravWithInactiveIdent = inactiveIdenter.flatMap { database.getAktivitetskrav(it) }
+                val oldPersonIdentList = inactiveIdenter.flatMap { personIdent -> database.getAktivitetskrav(personIdent).map { it.personIdent } }
```

Comment:
Det man ønsker å oppnå her er vel å sitte igjen med en liste med kun gamle identer som faktisk forekommer i databasen? 🤔 
Det kan evt. løses sånn:
`val oldPersonIdentList = inactiveIdenter.filter { database.getAllAktivitetskravByPersonident(it).isNotEmpty() }`